PkSolver/PkSolver.h

```diff
@@ -133,37 +133,76 @@ class LMCostFunction: public itk::MultipleValuedCostFunction
   MeasureType GetValue( const ParametersType & parameters) const
   {
     MeasureType measure(RangeDimension);
+	MeasureType measure2(RangeDimension);
+	MeasureType measure3(RangeDimension);
 
     ValueType Ktrans = parameters[0];
     ValueType Ve = parameters[1];
+	//Ktrans = .2;
+	//Ve = .1;
 
-    ArrayType VeTerm;
-    VeTerm = -Ktrans/Ve*Time;
-    ValueType deltaT = Time(1) - Time(0);
+	ArrayType VeTerm;
```

Comment:
@AndrewBeers can you fix the indentation to be consistent with the rest of the code?